### PR TITLE
task2: Serve SPA in AWS using serverless

### PR DESCRIPTION
**Pull request by SKolevatov**
**Task:** https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md
**Screenshot:**
image

**Done:** 22.10.2020 / deadline 24.10.2020
**Score:** 5 / 5

Link to CloudFront website: https://d3q52y5xysba04.cloudfront.net/
S3: http://my-first-app-rs.s3-website-us-east-1.amazonaws.com/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,34 +5,49 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
+    "build:deploy": "npm run build && npm run deploy:s3",
+    "client:deploy": "sls client deploy --no-config-change --no-policy-change --no-cors-change",
```
Comment (machulan) [inline package.json:8]: The script duplicates 'deploy:s3' script

File: package.json [hunk 2]
```diff
@@ -5,34 +5,49 @@
...
+    "client:deploy:nc": "npm run client:deploy -- --no-confirm",
+    "client:build:deploy": "npm run build && npm run client:deploy",
```
Comment (machulan) [inline package.json:10]: The script duplicates 'build:deploy' script

File: serverless.yml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
```
Comment (machulan) [inline serverless.yml:25]: You may add eu-west-1 as a region parameter here

**Comment by machulan** [comment]
Please, follow RS School convention for commit messages: https://docs.rs.school/#/git-convention

**Comment by machulan** [comment]
Please, follow RS School convention for PR description: https://docs.rs.school/#/stage2?id=Требования-к-pull-request-pr

**Comment by SKolevatov** [comment]
@nikinsta, thank you for your comments! Made the corresponding code changes.

**Review by ViktarYarmalovich — APPROVED** [review]

**Pull request merged**